pkg/interactive/interactive_client.go

```diff
@@ -587,15 +583,18 @@ func (c *InteractiveClient) queryCompleter(d prompt.Document) []prompt.Suggest {
 	if !c.isInitialised() {
 		return nil
 	}
+	if !cmdconfig.Viper().GetBool(constants.ArgAutoComplete) {
```

Comment:
This should probably be the first thing we do. Is there any reason to check for `c.isInitialised()` if `auto complete` is disabled anyway?